test/service/test_manifest.py

```diff
@@ -207,7 +207,9 @@ def to_json(records):
         # We write entities differently depending on debug so we test both cases
         for debug in (1, 0):
             with self.subTest(debug=debug):
-                with mock.patch.object(type(config), 'debug', debug):
+                patch_config = mock.patch.object(type(config), 'debug', debug)
+                patch_environ = mock.patch.dict(os.environ, {'AZUL_DRS_DOMAIN_NAME': 'test'})
+                with patch_config, patch_environ:
```

Comment:
This is tedious. Just nest two `with` statements. Patch the config object, not the environment because that's the precedent here. If, for some, reason, you have to use `patch.dict`, please explain and note that it takes keyword arguments.